### ISSUE TITLE
Tab Component

### ISSUE THREAD

**Issue by ernes7** [MetLifeLegalPlans/wui#82]
Tabs needs OnKeyDown Property or some sort of focus indicator.
The original focus indicator seems to have been replaced by a custom indicator.

From original issue:
 When you use the keyboard and navigate to the Summary tab, then a focus indicator should appear
 When you use the keyboard and navigate to the Help tab, then a focus indicator should appear
 The focus indicator should have a contrast of at least 3:1 versus the unfocused components
 The focus indicator should have padding of at least 2 CSS versus the unfocused components

Original Issues:
https://github.com/MetLifeLegalPlans/estate-planning/issues/2961
https://github.com/MetLifeLegalPlans/estate-planning/issues/2957
